### DISCUSSION TITLE
What is FlashFloppy?

### DISCUSSION THREAD

**Discussion by robrioja** [keirf/flashfloppy#742] [Q&A]
What does FlashFloppy actually do? I cannot find anything in the wiki that explains why I should use it. It simply says that it is a floppy emulator, but Gotek does that already. So why should I use FlashFloppy?
If I do use it, how do I write files to the USB stick? I read about "image" files, but found no explanation of what they are, or how to use them.  How do I copy files to the USB stick from my Windows PC? Do I need some special software?
Any answers will be appreciated.
Thank you.

**Comment by tenuke** [comment]
I suppose its mostly about compatibility. Gotek as it is, emulates PC
Floppydrive.
PC drives wont work with that list of hardware which you saw in the wiki.
Also you could not use floppy images with gotek if I remember correctly.
Flashfloppy firmware supports loads of different images..

You can just copy the files(images) to usb-stick just like that on a
windows PC or anything that
support bog standard FAT-formatted usb-drives. No special software is used.

"What is a floppy disk image?"
"A disk image is a file which is an exact and complete image of a floppy
disk.
The image contains information on the disk format and structure, such as
FAT,
boot sector, directories, and all files."

I dot know but maybe not everything is explained to ppl like they would be
five in wiki,
because this flashfloppy is not after all tried to be sold to anyone.
Its up to individuals to know that they might want some. Hobbyists usually
just understand
the basics I guess.

I have no idea if FF would benefit you, but its free to try and use.

p.s. Im just another random person, not developer of FF or anything like
that.


su 8. tammik. 2023 klo 5.20 robrioja ***@***.***) kirjoitti:

**Reply by robrioja** [reply]
Thank you for your information. Since the Gotek that I have works perfectly in all of my machines, I don't see a reason for using FlashFloppy. Unless, of course, FlashFloppy has some advantages that I don't see.  Currently, I am using the software from Ketron which allows me to read and write files to the 100 "floppies" in my USB stick without need for image files. So can you think of any real reason for switching to FlashFloppy?
Thank you very much for your help.

**Reply by keirf** [reply]
If you only need to emulate PC formatted disks and the management of files in the 100 partitions works well for you then there is really no point in changing out the firmware. 

In favour of disk images, they allow easy backup. The real power of FlashFloppy however is in emulating all the non PC disk formats.

**Reply by robrioja** [reply]
Thank you all for the information.
